STYLE.md

```diff
@@ -18,206 +18,150 @@ guidelines on some of the principles underlying the compiler design.
 Style Rules
 ===========
 
-Formatting
-----------
-
-### Line Length
+Ormolu
+------
 
-Maximum line length is *80 characters*.
+Futhark uses [Ormolu](https://github.com/tweag/ormolu/) to enforce consistency
+in formatting style. The style is checked by our CI, so you should make sure any
+pull requests comply with Ormolus formatting before trying to merge.
 
-### Indentation
+### Installation
 
-Tabs are illegal. Use spaces for indenting.  Indent your code blocks
-with *2 spaces*.  E.g, we write
+Installing Ormolu can be done through Cabal, Stack, Nix or the Arch Linux
+package manager. For instance, to install through Cabal, run the following
+command:
 
-```haskell
-foo x = do
-  case whatever x of
-    Just bar ->
-      ...
-    Nothing ->
-      ...
+```
+cabal install ormolu
 ```
 
-It is okay to use a different number of spaces for alignment with
-other syntactic structures, e.g:
+If you're running Nix or NixOS, you can just use `nix-shell` to enter a
+development environment with ormolu already installed.
 
-```haskell
-case foo of Just x  -> do a
-                          b
-            Nothing -> do c
-                          d
+### Basic Usage
+
+The following command formats a single file:
+
+```
+ormolu -i FILE.hs
 ```
 
-Tasteful alignment can made code easier to read, but do not go
-overboard.  The functions `inputs` and `lambda` in
-`Futhark.Analysis.HORepresentation.SOAC` are a good example of
-pointless use of alignment, and are retained to serve as a reminder of
-this.
+This command can be used to format all Haskell files in Futhark, while
+checking that the formatting is idempotent:
 
-If in doubt, don't align.  Spurious alignment makes the code feel weird
-and off-key, and it can be remarkably ugly when not maintained.  Alignment
-in expressions is usually a bad idea.
+```
+find src unittests -name '*.hs' -print -execdir ormolu --mode inplace --check-idempotence {} +
```

Comment:
Can you put a script in `tools/` that does this, like we have for style checking?